guides/source/developers/adjustments/overview.html.md

```diff
@@ -109,13 +111,12 @@ For example, you can find all of the adjustments with an `eligible` value of
 - `charge`: Returns adjustments with a positive value.
 - `credit`: Returns adjustments with a negative value.
 - `is_included`: Returns adjustments that are included in the object's price.
-   Typically, only value-added tax adjustments have this value.
+   Typically, only [value-added tax][value-added tax] adjustments have this
+   value.
 - `additional`: Adjustments which modify the object's price. The default for all
   adjustments.
 
-<!-- TODO:
-  Add link to taxation documentation to `is_included` item in the list.
--->
+[value-added tax]: ../taxation/overview.html#sales-tax-and-value-added-tax
```

Comment:
We have this entity cited into `Tax adjustments` section as well (line 64). What about adding the link for that text as well?

Reply:
Updated Thank you 😄